### DISCUSSION TITLE
Linking dataset run items with existing callback handler

### DISCUSSION THREAD

**Discussion by subandwho** [langfuse/langfuse#1445] [Support]
Hi everyone, currently trying to log a few custom scores but am seeing each item from my dataset get logged seperately,
Not sure when i am setting the same trace_id why the issue must persist.

I am using langchain_handler for each item of the dataset. Did not find anything resolving this issue on the docs. Hope somebody can help.

TLDR: Need to log scores + IO, user_id etc together as one via trace_id or other relevant identifier

**Comment by ladislasdellinger** [comment]
Hi @subandwho,

I'm not sure to understand what you're trying to do.
What do you mean by "logged separately" ?

**Reply by marcklingen** [reply]
Also curious to get more details on this
